exercise1/exercise1/main.cpp

```diff
@@ -0,0 +1,12 @@
+#include <iostream>
+//Exercise 1 - Hello World
+//Write a program that will output �Hello World!� in the first line, then your nameand something about yourself in the second line
+
+
+int main()
+{
+	std::cout << "hello word�" << std::endl;
```

Comment:
Code is good, but you're not printing the correct text, be mindful of your code and the instructions :)